css/style.css

```diff
@@ -0,0 +1,51 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  background-color: #eee;
+}
+
+.topnav {
+  overflow: hidden;
+  background-color: #ddd;
+}
+
+.topnav a {
+  float: left;
+  color: #f2f2f2;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 17px;
+}
+
+.topnav a:hover {
+  background-color: #ddd;
+  color: black;
+}
+
+.topnav a.active {
+  background-color: #04aa6d;
+  color: white;
+}
+
+.logo-image {
+  width: 150px;
+  height: 46px;
+  border-radius: 50%;
+  overflow: hidden;
+  margin-top: 6px;
+  margin-left: 25px;
+}
+
+.header nav {
+  margin-top: 10px;
+  margin-bottom: 10px;
+  width: 100%;
+  display: flex;
+}

```

Comment:
I recommend you leave the line after finishing your code. Sometimes it causes some linter errors and you would probably be stuck in with it.😄